File: MATEMÁTICA/1247/main.rb
# frozen_string_literal:true

def can_catch(d, vf, vg)
  sg = vg * 12 / vf
  sg * sg >= d * d + 12 * 12
end

while (line = gets)
  d, vf, vg = line.split.map(&:to_i)
  puts can_catch(d, vf, vg) ? 'S' : 'N'
end

File: MATEMÁTICA/1247/main.js
function canCatch(d, vf, vg) {
    let sg = vg * 12 / vf;
    return sg * sg >= d * d + 12 * 12;
}

let input = require('fs').readFileSync('/dev/stdin', 'utf-8');
let lines = input.split('\n');
for (let line of lines) {
    if (line) {
        let [d, vf, vg] = line.split(' ').map(Number);
        console.log(canCatch(d, vf, vg) ? 'S' : 'N')
    }
}